src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Performance/Microsoft.Azure.ServiceBus.Performance.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Is there any reason we shouldn't have this project end in ".Tests" like other test projects?

Reply:
Because it's a manually-run performance test, not a functional or unit test.  The MSBuild files in this repo already have special logic for projects with end in `.Performance`.

Reply:
Sounds reasonable. 

I missed https://github.com/Azure/azure-sdk-for-net/pull/5892 so didn't see the ".Performance" convention.